<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Booking_new</class>
 <widget class="QDialog" name="Booking_new">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление бронирования</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>481</height>
    </rect>
   </property>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>128</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>По</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="dateEdit_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>341</width>
      <height>281</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>339</width>
       <height>279</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Условия фильрации:</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>440</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_5">
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Выберите номер </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pb_booking">
       <property name="text">
        <string>Забронировать</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Reservations_table" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>501</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>601</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Шаг 1. Введите даты и выберите номер</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_addRow">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Reservations_table</class>
   <extends>QTableWidget</extends>
   <header>reservations/reservations_table.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
